libs/langchain/langchain/vectorstores/chroma.py

```diff
@@ -517,19 +517,30 @@ def update_document(self, document_id: str, document: Document) -> None:
             document_id (str): ID of the document to update.
             document (Document): Document to update.
         """
-        text = document.page_content
-        metadata = document.metadata
+        return self.update_document_batch([document_id], [document])
+
+    def update_document_batch(
```

Comment:
A few questions:

Looking for green light on the following renamings:

1. update_document_batch -> update_documents or upsert_documents (to match add_documents)
2. document_ids -> `ids` to match naming in other methods of base class

What are the semantics of the update if the id is invalid? Is this upsert or does it raise an exception?

I'd like to standardize this in the base store

Reply:
Chroma supports `upsert` and `update` - `update` with a non-existent ID will log an error- it does not raise an exception.

Reply:
Re: naming things - that is up to langchain I would say - I'm not sure what their vernacular is. :) 

I was simply clarifying your question.

Reply:
@eyurtsev I believe that update_document_batch -> update_documents is more related name with "update_document" in a user point of view. 

1) If it is okay I can rename update_document_batch -> update_documents and document_ids -> ids
2) Should we raise an exception for invalid id (It can be checked by `get` function)? Or log an error is enough (I think it is enough because it works like try, excep -> if an id is invalid log an error _Update of nonexisting embedding ID: 3_
and continue the process ?

I put an example of a working of update_document on the below:

<img width="1254" alt="Screenshot 2023-08-24 at 19 21 36" src="https://github.com/langchain-ai/langchain/assets/19657350/15ef95af-16f0-4ec7-8595-649ea7feea2c">